invoice.go

```diff
@@ -122,6 +129,27 @@ func NewInv(inv *bill.Invoice) *Inv {
 				} else if rate.Key == tax.RateSuperReduced {
 					Inv.SuperReducedRateNetSale = rate.Base.Rescale(cu).String()
 					Inv.SuperReducedRateTax = rate.Amount.Rescale(cu).String()
+				} else if rate.Key == tax.RateSpecial && rate.Ext.Has(pl.ExtKeyKSeFVATSpecial) {
+					if rate.Ext[pl.ExtKeyKSeFVATSpecial].String() == "taxi" {
+						Inv.TaxiRateNetSale = rate.Base.Rescale(cu).String()
+						Inv.TaxiRateTax = rate.Amount.Rescale(cu).String()
+					}
+				} else if rate.Key == tax.RateZero && rate.Ext.Has(pl.ExtKeyKSeFVATRegion) {
+					if rate.Ext[pl.ExtKeyKSeFVATRegion].String() == "domestic" {
+						Inv.DomesticZeroTaxNetSale = rate.Base.Rescale(cu).String()
+					} else if rate.Ext[pl.ExtKeyKSeFVATRegion].String() == "EU" {
```

Comment:
I think it could be changed to something like this:
```go
for _, rate := range cat.Rates {
	if rate.Percent == nil {
		continue
	}

	switch rate.Key {
	case tax.RateStandard:
	case tax.RateReduced:
	case tax.RateSuperReduced:
	case tax.RateSpecial:
	case tax.RateZero:
	case tax.RateExempt:
	case pl.TaxRateNotPursuant:
	}
}
```

And each case could be a separate function. This will have a positive impact on readability.